snuba/snuba_migrations/metrics_summaries/0001_metrics_summaries_create_table.py

```diff
@@ -0,0 +1,79 @@
+from typing import List, Sequence
+
+from snuba.clickhouse.columns import UUID, Column, DateTime, Nested, String, UInt
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations, table_engines
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+from snuba.migrations.operations import OperationTarget, SqlOperation
+from snuba.utils.schemas import Float
+
+storage_set_name = StorageSetKey.METRICS_SUMMARIES
+local_table_name = "metrics_summaries_local"
+dist_table_name = "metrics_summaries_dist"
+
+columns: List[Column[Modifiers]] = [
+    # ids
+    Column("project_id", UInt(64)),
+    Column("span_id", UInt(64)),
+    Column("trace_id", UUID()),
+    # metrics summary
+    Column("metric_name", String(Modifiers(low_cardinality=True))),
+    Column("min", Float(64)),
+    Column("max", Float(64)),
+    Column("sum", Float(64)),
+    Column("count", Float(64)),
```

Comment:
Nit: Not sure under what circumstances can count be a float. If it can't be a float then you can probably change it to int. Takes lesser space to store.